includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -417,6 +417,16 @@ public function get_title() {
 		return apply_filters( "wpo_wcpdf_{$this->slug}_title", $this->title, $this );
 	}
 
+	public function get_number_title() {
+		$number_title = sprintf( __( '%s Number', 'woocommerce-pdf-invoices-packing-slips' ), $this->title );
```

Comment:
this seems problematic with translations to me. While it's nice as a fallback, we definitely also need the 'fixed' strings inside each document class.

Reply:
Yes, must follow this: https://github.com/wpovernight/woocommerce-pdf-ips-pro/pull/59/commits/cab1d3cabaad1078e10cf14fb1eedebb6b62db39